scripts/eachdist.py

```diff
@@ -236,6 +236,13 @@ def setup_instparser(instparser):
         "releaseargs", nargs=argparse.REMAINDER, help=extraargs_help("pytest")
     )
 
+    patchreleaseparser = subparsers.add_parser(
+        "update_patch_versions",
+        help="Updates version numbers during patch release, used by maintainers and CI",
+    )
+    patchreleaseparser.set_defaults(func=patch_release_args)
+    patchreleaseparser.add_argument("--versions", required=True)
```

Comment:
instead of having a list like this, can we just add a separate argument for each of the versions? IIUC 
```suggestion
    patchreleaseparser.add_argument("--stable_version", required=True)
    patchreleaseparser.add_argument("--unstable_version", required=True)
    patchreleaseparser.add_argument("--stable_version_prev", required=True)
    patchreleaseparser.add_argument("--unstable_version_prev", required=True)

```